packages/shared/__tests__/split-value-test.js

```diff
@@ -44,4 +44,13 @@ describe('Ensure CSS values are split correctly', () => {
       splitValue('calc((100% - 50px) * 0.5) var(--rightpadding, 20px)'),
     ).toEqual(['calc((100% - 50px) * 0.5)', 'var(--rightpadding,20px)']);
   });
+
+  test('Splits a string of values with slash notation appropriately.', () => {
+    expect(splitValue('1px/2px 3px 4px 5px')).toEqual([
+      '1px/2px',
+      '3px',
+      '4px',
+      '5px',
+    ]);
+  });
 });
```

Comment:
This is not how border-radius values are expanded. The correct expansion should be:


```suggestion

  test('Splits a string of values with slash notation appropriately.', () => {
    expect(splitValue('1px/2px 3px 4px 5px')).toEqual([
      '1px 2px',
      '1px 3px',
      '1px 4px',
      '1px 5px',
    ]);
  });
});
```

See https://developer.mozilla.org/en-US/docs/Web/CSS/border-radius
which mentions:

```css
border-radius: 4px 3px 6px / 2px 4px;

/* It is equivalent to: */
border-top-left-radius: 4px 2px;
border-top-right-radius: 3px 4px;
border-bottom-right-radius: 6px 2px;
border-bottom-left-radius: 3px 4px;
```

Reply:
I misunderstood the usage of `/` in `border-radius`. I will review the specification and correct the code accordingly.